build/AzurePipelinesTemplates/MUX-RunHelixTests-Job.yml

```diff
@@ -120,7 +126,30 @@ jobs:
       
   - task: PublishBuildArtifacts@1
     displayName: 'Publish ${{parameters.name}} binlog'
-    condition: and(succeededOrFailed(), eq(variables['useBuildOutputFromBuildId'],''))
+    condition: succeededOrFailed()
     inputs:
       PathtoPublish: $(Build.SourcesDirectory)/${{parameters.name}}.$(buildPlatform).$(buildConfiguration).binlog
-      artifactName: 'drop' 
\ No newline at end of file
+      artifactName: 'drop'
+      
+- job: ProcessTestResults
```

Comment:
I'm curious: Why is this a separate Job?

Reply:
Making it a separate job dependent on the previous job is the easiest way to ensure that it won't run until all of the test runs are complete, since the matrix causes us to run two separate jobs concurrently for x86 and x64.